radis/lbl/calc.py

```diff
@@ -384,8 +391,30 @@ def calc_spectrum(
     """
 
     # Check inputs
-
-    # ... wavelengths / wavenumbers
+    if "molecule" in kwargs:
+        if species is not None:
+            if species != kwargs['molecule']:
+                raise Exception("Both `molecule` and `species` arguments have been given and aren't equal, but `molecule` is deprecated and `species` is just its replacement.")
+        species = molecule = kwargs['molecule']
+        warn(
+            DeprecationWarning(
+                "`molecule` is deprected - use `species` instead"
+            )
+        )
```

Comment:
No, our coding philosophy should be "Better Crash Than Hide" 
If users added a different molecule and species; then something is wrong. Let's raise an explicit error rather than try to fix it behind their back.

Reply:
Agreed with Erwan. If the user specifies both, just crash with an explicit error. 
Also, I'm not sure that we will depreciate the "molecule" input in the near future, so I would simply ask the user to pick one or the other.

Reply:
> No, our coding philosophy should be "Better Crash Than Hide" If users added a different molecule and species; then something is wrong. Let's raise an explicit error rather than try to fix it behind their back.

That's precisely what we're doing here: `if species != kwargs['molecule']` then raise an exception, otherwise if they are equal, then it doesn't cause a problem per se, it's just that giving both arguments is redundant so the `DeprecationWarning` is given.

Or should we be raising an exception even when both are equal?

Reply:
Sorry I was too quick in reading your code. This is exactly what we should do. Keep it as is ! You can resolve this conversation